docs/core/integration-quality-scale/rules/iqs011.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "IQS011 - Entities have a unique ID"
+---
+
+IQS011 is as follows:
+
+Entities have a unique ID
+
+## Reasoning
+
+In the past, entities used to be volatile.
```

Comment:
I don't think volatile is the right word. The speed of change isn't what's important here. The important thing is that we didn't store the customizations done by the user and we couldn't track the entities.